server/plugin/taskLinkPreview.go

```diff
@@ -154,3 +154,42 @@ func (p *Plugin) PostBuildDetailsPreview(linkData []string, link, userID, channe
 
 	return post, ""
 }
+
+func (p *Plugin) PostReleaseDetailsPreview(linkData []string, link, userID, channelID string) (*model.Post, string) {
+	organization := linkData[3]
+	project := linkData[4]
+	releaseID := strings.Split(linkData[5], "&")[1][10:]
+	releaseDetails, _, err := p.Client.GetReleaseDetails(organization, project, releaseID, userID)
+	if err != nil {
+		return nil, ""
+	}
+
+	post := &model.Post{
+		UserId:    userID,
+		ChannelId: channelID,
+	}
+	environments := p.getPipelineReleaseEnvironmentList(releaseDetails.Environments)
```

Comment:
```suggestion
	}
	
	environments := p.getPipelineReleaseEnvironmentList(releaseDetails.Environments)
```

Reply:
can you let me know on what basis you decide to add a line break?

Reply:
I usually add a line break at the end of a block